config/environments/test.rb

```diff
@@ -39,4 +39,7 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # Search options for member
+  config.algolia_search = false
```

Comment:
How will this get tested? :)

Reply:
Such a good question 🐣 🐔 :)    It's one of the reasons I want to get the ENV issue (wrong Algolia API key running locally).  That would be step 1.  I'm wondering how or if it is recommended to find an automated way of flipping that switch when tests are run?  Is locally testing enough?  Or CI should test too?